cli/flox/src/commands/mod.rs

```diff
@@ -354,7 +354,22 @@ enum AdditionalCommands {
     ),
     #[bpaf(command, hide)]
     Update(#[bpaf(external(environment::update))] environment::Update),
-    #[bpaf(command, hide)]
+    #[bpaf(command, hide, header(indoc! {"
+        When no arguments are specified, all packages in the environment are upgraded.
+        
+        
+        
+        Packages to upgrade can be specified by either group name or, if a package is
+        not in a group with any other packages, it may be specified by ID. If the
+        specified argument is both a group name and a package ID, only the group is
+        upgraded.
+        
+        
+        
+        Packages without a specified group in the manifest can be upgraded by passing
+        'toplevel' as the group name.
+    "}))]
+    /// Upgrade packages in an environment
```

Comment:
```suggestion
    #[bpaf(command, hide, descr("Upgrade packages in an environment"),header(indoc! {"
        When no arguments are specified, all packages in the environment are upgraded.\n\n\n

        Packages to upgrade can be specified by either group name or, if a package is
        not in a group with any other packages, it may be specified by ID. If the
        specified argument is both a group name and a package ID, only the group is
        upgraded.\n\n
        
        Packages without a specified group in the manifest can be upgraded by passing
        'toplevel' as the group name.
    "}))]

```

Reply:
That would indent just the second paragraph which I think looks weird, so I used `\n\n` instead of `\n\n\n`. Your suggestion:
```
When no arguments are specified, all packages in the environment are upgraded.

 Packages to upgrade can be specified by either group name or, if a package is not in a group with any
other packages, it may be specified by ID. If the specified argument is both a group name and a package
ID, only the group is upgraded.

Packages without a specified group in the manifest can be upgraded by passing 'toplevel' as the group
name.
```

What I pushed:
```
When no arguments are specified, all packages in the environment are upgraded.

Packages to upgrade can be specified by either group name or, if a package is not in a group with any
other packages, it may be specified by ID. If the specified argument is both a group name and a package
ID, only the group is upgraded.

Packages without a specified group in the manifest can be upgraded by passing 'toplevel' as the group
name.
```

Reply:
Ah yeah thanks, well spotted
The review was a bit improvised not seeing the output.